packages/ag-charts-website/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": "../../tsconfig.base.json",
+  "include": ["./src/content/docs/**/_examples/**/*.ts", "./src/content/gallery/_examples/**/*.ts"],
```

Comment:
This excludes the rest of the website package, however there are currently 800 ts errors in that. But none of those are used in the library so not a big deal.

This does though hinder some behaviour in vscode around our usage of `import foo from @design-system/foo` etc. Not a major issue and we can fix up all the ts errors in the website at some point later.